### PR TITLE
Replaced Razor syntax with TagHelpers

### PR DESCRIPTION
**Pull request by rafsanulhasan**
@thiennn In this PR, I replaced all:
- [x] `@Html.Partial` and `@Html.PartialAsync`
- [x] `@Html.RenderPartial` and `@Html.RenderPartialAsync`

with **`<partial />`** TagHelper

Removed indentation issues from PR #549

### CODE REVIEW

**Comment by justindongqiang** [comment]
@rafsanulhasan good work.

**Comment by rafsanulhasan** [comment]
@justindongqiang Thanks!
@thiennn Would you please review this?

**Comment by justindongqiang** [comment]
I recommend to use ExpandViewLocations to simplify view path.

**Comment by rafsanulhasan** [comment]
I think its not needed because I have another branch RCL that will introduce Razor Class Library. There will be no need to copy module Views using Gulp. There will be no use of view location expander. By using Razor Class LiBrary (RCL), the view of subsequent modules will be automatically derived to its location (eg. src\Modules\SimplCommerce.zModule.Core\Views directory will be derived to src\SimplCommerce.WebHost\Views directory). To know more visit [here](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/ui-class?view=aspnetcore-2.1&tabs=visual-studio), FYI visit [here](https://gunnarpeipman.com/aspnet/razor-class-library-mvc/)

**Comment by justindongqiang** [comment]
@rafsanulhasan great and I will look at them, thanks for sharing.

File: src/SimplCommerce.WebHost/appsettings.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿{
   "ConnectionStrings": {
```
Comment (hishamco) [inline src/SimplCommerce.WebHost/appsettings.json]: Why you change the ConnectionString?!!

File: src/SimplCommerce.WebHost/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -106,7 +106,7 @@
     <script simpl-append-version="true" src="~/modules/product-comparison/product-comparison.js"></script>
     <script simpl-append-version="true" src="~/modules/wish-list/wishlist.js"></script>
     @RenderSection("scripts", required: false)
-
-    @await Html.PartialAsync("_AnalyticsScript")
+
```
Comment (hishamco) [inline src/SimplCommerce.WebHost/Views/Shared/_Layout.cshtml]: Whitespace

File: src/SimplCommerce.WebHost/Views/Shared/_Layout.cshtml [hunk 3]
```diff
@@ -106,7 +106,7 @@
     <script simpl-append-version="true" src="~/modules/product-comparison/product-comparison.js"></script>
     <script simpl-append-version="true" src="~/modules/wish-list/wishlist.js"></script>
     @RenderSection("scripts", required: false)
-
-    @await Html.PartialAsync("_AnalyticsScript")
+	
+    <partial name="~/Views/Shared/_AnalyticsScript.cshtml" />
```
Comment (hishamco) [inline src/SimplCommerce.WebHost/Views/Shared/_Layout.cshtml]: Are we need a full relative path?!!

File: src/Modules/SimplCommerce.Module.Comments/Views/Comment/List.cshtml [hunk 4]
```diff
@@ -73,7 +74,10 @@
                         <div>
                             <button class="btn-add-review-reply btn btn-link">@Localizer["Add Reply"]</button>
                             <div class="add-review-reply d-none">
-                                @Html.Partial("/Modules/SimplCommerce.Module.Comments/Views/Comment/_ReplyForm.cshtml", new CommentForm { ParentId = comment.Id, EntityTypeId = Model.EntityTypeId, EntityId = Model.EntityId })
+                                @{
```
Comment (hishamco) [inline src/Modules/SimplCommerce.Module.Comments/Views/Comment/List.cshtml]: Why you didn't pass the model as it was before?

File: src/SimplCommerce.WebHost/Views/Shared/_Layout.cshtml (hunk 3 above)
Comment (rafsanulhasan) [reply src/SimplCommerce.WebHost/Views/Shared/_Layout.cshtml]: Yes, we need. But when we'll use RCL, we will probably don't need it

File: src/Modules/SimplCommerce.Module.Comments/Views/Comment/List.cshtml (hunk 4 above)
Comment (rafsanulhasan) [reply src/Modules/SimplCommerce.Module.Comments/Views/Comment/List.cshtml]: Yes I could have done this.

**Comment by rafsanulhasan** [comment]
what about this thienn? can this be merged or should be closed?

**Comment by rafsanulhasan** [comment]
@thienn what about this? can we merge it?

**Comment by hishamco** [comment]
Why this is closed?!!

**Comment by rafsanulhasan** [comment]
@hishamco this is kind of messy and I didn't get any response from @thiennn though. so I closed it.

**Comment by hishamco** [comment]
Sometimes PR needs a review, test .. etc. I had some PR for more than year, but that doesn't mean it doesn't have a value, may be it's not the right time to accept it

**Comment by rafsanulhasan** [comment]
I have never used GitHub as an SCCM and project management tools before I started contributing the open-source community few weeks back. Prior to that, I have been generally using TFS for Application Life-cycle Management (ALM) which includes SCCM, Project Management, Process Management and so on...

Before I started contributing open-source projects, I just worked with on-premises teams those usually had a specific time frame (In Agile or Scrum), so that developer can know when will their work be reviewed but unlike I did't get any reply from the lead @thiennn that this PR would possibly be reviewed soon. So I had no idea about what to do with the PR. Additionally, I had another work to do which is nearly dependent to this PR. So, until this PR is reviewed and merged, I was stuck. 

I did the RCL stuff in PR #586 which includes this work though.

Anyway, thanks for the suggestion because there is no end to learning and I've to learn how to deal with open-source community as well as open-sourced projects (hosted in GitHub or BitBucket) elaborately.

**Pull request closed without merging**